src/github.com/matrix-org/dendrite/clientapi/routing/register.go

```diff
@@ -376,6 +422,62 @@ func isValidMacLogin(
 	return hmac.Equal(givenMac, expectedMAC), nil
 }
 
+const letterBytes = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
+const (
+	letterIdxBits = 6                    // 6 bits to represent a letter index
+	letterIdxMask = 1<<letterIdxBits - 1 // All 1-bits, as many as letterIdxBits
+	letterIdxMax  = 63 / letterIdxBits   // # of letter indices fitting in 63 bits
+)
+
+var src = rand.NewSource(time.Now().UnixNano())
+
+// RandString returns a random string of characters with a given length.
+// Do note that it is not thread-safe in its current form.
+// https://stackoverflow.com/a/31832326
+func RandString(n int) string {
```

Comment:
`util.RandomString` is already a thing

Reply:
Cool, was hoping we'd have something already :)